ip/draft-ietf-detnet-ip.xml

```diff
@@ -277,14 +284,13 @@
       <t>
         <xref target="fig_ip_detnet_simple"/> illustrates a DetNet enabled IP
         network.  The DetNet enabled end systems originate IP encapsulated
-        traffic that is identified as DetNet flows, relay nodes understand the
+        traffic those are identified within the DetNet domain as DetNet flows, 
+		relay nodes understand the
         forwarding requirements of the DetNet flow and ensure that node,
         interface and sub-network resources are allocated to ensure DetNet
         service requirements. The dotted line around the Service component of
         the Relay Nodes indicates that the transit routers are DetNet service
         aware but do not perform any DetNet service sub-layer function, e.g., PREOF.
-        IEEE 802.1 TSN is an example sub-network type which can provide support
-        for DetNet flows and service.
```

Comment:
What is the value of deleting this informative sentence?

Reply:
It is redundant. In the next sentence we list various sub-network technologies including TSN.

Reply:
okay